src/shared/codelens/codeLensUtils.ts

```diff
@@ -123,6 +126,81 @@ function makeAddCodeSamDebugCodeLens(
     return new vscode.CodeLens(range, command)
 }
 
+/**
+ * Tied to the AWS.addSamDebugConfig command: lets a user create a config tied to a handler via command instead of codelens
+ * Renders a quick pick using the first 200 characters of the first line of the funciton declaration + function line number
+ * @param document Curr document
+ * @param lenses Codelenses returned via CodeLensProvider, which we extract the information from
+ */
+export async function invokeCodeLensCommandPalette(
+    document: Pick<vscode.TextDocument, 'getText'>,
+    lenses: vscode.CodeLens[]
+): Promise<void> {
+    const handlers: (vscode.QuickPickItem & { lens?: vscode.CodeLens })[] = lenses
+        .filter(lens => {
+            // remove codelenses that go to the invoker UI
+            // maybe move this into the workflow at some point (drop down to one)
+            return (
+                lens &&
+                lens.command &&
+                lens.command.arguments &&
+                lens.command.arguments.length === 3 &&
+                lens.command.arguments![2] !== true
+            )
+        })
+        .map(lens => {
+            return {
+                // lens is currently pulling the entire function, not just the declaration
+                label: document.getText(
+                    new vscode.Range(
+                        lens.range.start,
+                        new vscode.Position(lens.range.start.line, lens.range.start.character + 200)
```

Comment:
Might as well make the 200 a constant, `HANDLER_MAX_SIZE` maybe. Does this work for multi-line function declarations? Haven't looked to much into `vscode.Position` but I imagine it would.

Reply:
Multiline doesn't render nicely since it renders it all into one line (our codelens ranges actually aren't great in that we flag the ENTIRE function, at least for JS).

I figure this is good enough for now, this shows the line that it would render on normally. Can move to a const.